app/services/attribute_asserter.rb

```diff
@@ -44,9 +44,11 @@ def default_attrs
   end
 
   def add_bundle(attrs)
+    use_ascii = bundle.include?(:ascii)
```

Comment:
If `ascii?` gets called inside the `bundle.each` loop, it would be more efficient if it were memoized.

if it gets called outside the loop, I'm not sure what the benefit of Yet Another Method would be

```ruby
use_ascii = ascii?
# vs
use_ascii = bundle.include?(:ascii)
```